src/Tribe/Customizer/Front_Page_View.php

```diff
@@ -45,14 +45,27 @@ public function setup() {
 	 */
 	public function add_events_page_option( $output, $r, $pages ) {
```

Comment:
Can we use a better variable name for `$r`?

Reply:
that's the name of the var in the core function so i just kept it consistent, which i think is the right move -- thoughts?

Reply:
Change it to something more read-able if we can.